### DISCUSSION TITLE
How to make python variables visible in Explorer?

### DISCUSSION THREAD

**Discussion by victormeldrew** [posit-dev/positron#4349] [Q&A]
I have a Python script and simply cannot figure out how to make variables appear in Data Explorer when debugging (as one can with RStudio). I can only see them in the Debug Variables window, which is not sufficient for my needs. Is there a shortcut or setting I'm missing? Many thanks

**Comment by seeM** [comment]
Thanks for the feedback @victormeldrew! It's not currently possible to use features of Positron that require communication with the kernel while debugging. I've filed this as an enhancement request: https://github.com/posit-dev/positron/issues/4350.

As a workaround, is it possible to run subsections of your code in the console?

**Reply by victormeldrew** [reply]
Many thanks for the info. I'lll give that workaround a try - nice idea
